setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="DB12",
+    description="DIRAC Benchmark 2012",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/pypa/DIRACGrid/DB12",
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
```

Comment:
According to [this README](https://github.com/pypa/setuptools_scm#setuppy-usage-deprecated): `setuptools_scm` should be placed in the `pyproject.toml` rather than in `setup.py` (deprecated).

Reply:
I tried removing it and placing it in pyproject.toml but the package doesn't get uploaded and it raises an error